UnchainedLauncherCoreTests/src/API/ServerBrowserTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using UnchainedLauncher.Core.API;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnchainedLauncher.Core.JsonModels.Metadata.V3;
+using Octokit;
+
+namespace UnchainedLauncher.Core.API.Tests
+{
+    public class MockServerBrowser : IServerBrowser
```

Comment:
This and other mocks should probably go in some kind of API/Mocks.cs file, since they're accessed in more than one file

Reply:
fair enough